provisioning/ansible/attacker_playbook.yml

```diff
@@ -19,6 +19,18 @@
         line: '127.0.1.1	attacker'
         state: absent
 
+  tasks:
+    - name: Clone grc
+      git:
+        repo: https://github.com/garabik/grc.git
+        dest: /root/grc
+        clone: yes
+        version: devel
+    - name: Install grc
+      shell:
+        chdir: /root/grc
+        cmd: ./install.sh
+        creates: /usr/local/bin/grc
```

Comment:
Aside from what Maspital wrote, I'm not quite happy with installing software via a shell script from an external repo. This is error-prone in case the repo or the script change, which can crash the whole SOCBED setup. Isn't there a package that could be used instead? If not, I would rather refrain from adding this feature.

Reply:
Sure, the installation can also be accomplished via `apt`. I will change the installation accordingly.